sycl/include/CL/sycl/device_triple.hpp

```diff
@@ -0,0 +1,91 @@
+//==-------------- triple.hpp - SYCL device triple descripter --------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <CL/sycl/backend_types.hpp>
+#include <CL/sycl/detail/defines.hpp>
+#include <CL/sycl/info/info_desc.hpp>
+
+#include <iostream>
+#include <string>
+
+__SYCL_INLINE_NAMESPACE(cl) {
+namespace sycl {
+
+class device_triple {
```

Comment:
Shouldn't we put this class to the detail or ONEAPI namespace? @jbrodman

Reply:
since no response from @jbrodman, I will put this in the 'detail' namespace. 
If necessary I will change it later accordingly.

Reply:
Done